parm/config/config.fcst

```diff
@@ -214,17 +215,11 @@ if [[ "$OUTPUT_FILE" == "netcdf" ]]; then
             export ichunk3d=$((4*RESTILE)) 
             export jchunk3d=$((2*RESTILE))
             export kchunk3d=1
-        else
-            export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
         fi
     fi
-    if [[ "$machine" == "HERA" ]]; then
-        export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
-    fi
     if [[ "$machine" == "ORION" ]]; then
-        export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf_parallel' "
-        if [ $RESTILE -le 192 ]; then
-            export OUTPUT_FILETYPES=" 'netcdf_parallel' 'netcdf' "
+        if [ $RESTILE -gt 192 ]; then
```

Comment:
Do we need this `if` statement?  Why not just use `netcdf_parallel` for both atm and sfc files for all resolutions?
Is there a problem?